docs/developing.md

```diff
@@ -122,6 +122,46 @@ kind delete cluster --name hive
 ./hack/create-kind-cluster.sh hive
 ```
 
+## Deploying with kind with Podman (EXPERIMENTAL)
+
+The process for running kind with Podman is similar to the Docker environment with some key differences. You will of course need Podman installed and Buildah as well. Last tested with kind v0.9.0.
+
+Create the kind cluster and deploy a local container registry.
+
+```bash
+./hack/create-kind-podman-cluster.sh hive
+```
+
+This will start up a kind cluster and a container registry under the root user (running rootless does not currently work). You should now have a kubeconfig in ~/.kube/hive.kube that can be used to interact with the kind cluster.
+
+You should now see a kind cluster and a container registry running under Podman under root.
+
+```bash
+$ sudo podman ps -a
+CONTAINER ID  IMAGE                                                                                           COMMAND               CREATED     STATUS         PORTS                      NAMES
+7a7ef66cdc70  docker.io/kindest/node@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600                        3 days ago  Up 3 days ago  127.0.0.1:38191->6443/tcp  hive-control-plane
+148e174d9634  docker.io/library/registry:2                                                                    /etc/docker/regis...  3 days ago  Up 3 days ago  0.0.0.0:5000->5000/tcp     kind-podman-registry
+```
+
+Build hive and push to the local container registry.
+
+```bash
+IMG=localhost:5000/hive:latest make buildah-dev-push
+```
+
+You can now use the regular `make deploy` and the certificate approval scripts (after installing the necessary `cfssl` tools) as normal.
```

Comment:
Is it a good idea to also add explicit commands for `make deploy`, creating certs  and to install `cfssljson` and `cfssl` here? Sometimes user may skip 'kind with docker' and directly jump to 'kind with podman' section of this doc.